### PR TITLE
Update figures to date

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

- Added new versions of the figures now that we are done with all panels except survival panels. I left those off for now. 
- I updated the supp figures legend text and updated the text within the results to refer to the proper figures and panels.
- I did not add new results (eg immune deconv/heatmap hypermutator discussion) here - will do on another stacked PR. Wanted to get these figures updated first so it is easier to review the next updates in text.
- also removed all supp figure embedded links since they do not number properly
 

### Issue

NA

#### Which areas should receive a particularly close look?
check whether this looks good!

#### Is there anything that you want to discuss further?
no

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: content/07.supplemental.md [hunk 1]
```diff
@@ -11,8 +11,10 @@ Figure created with biorender.com.
 **Figure S2. Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.**  
 Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. 
 Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. 
-Upset plot (F) showing overlap of variant calls. 
-Cumulative distribution TMB plots for PBTA (G) and TCGA (H)tumors using consensus SNV calls.
+Upset plot (F) showing overlap of variant calls.
+Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same samples (N = ).
+<!-- TODO: come back to this N. -->
```
Comment (sjspielman) [inline content/07.supplemental.md:16]: I just opened this PR to track `N` more clearly: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1296

File: content/07.supplemental.md [hunk 2]
```diff
@@ -42,4 +41,6 @@ Table of TMB, eight CNS mutational signatures, and chromothripsis events per sam
 
 **Table S3. Related to Figures 4 and 5.** Table of _TP53_ scores, telomerase EXTEND scores, and quanTIseq immune scores.
 
+**Table S4. Related to Figure 1.** Table of all software used for the OpenPBTA project.
```
Comment (sjspielman) [inline content/07.supplemental.md:44]:
```suggestion
**Table S4. Related to Figure 1.** Table of all software and their respective versions used for the OpenPBTA project.
```

**Review by sjspielman — APPROVED** [review]
This all looks good! You can add in the lancet N's to this legend at your discretion, given this https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1296. If during review we remove the subtitle from that figure and just print out the N's, we definitely want to add this into the legend.

File: content/07.supplemental.md (hunk 1 above)
Comment (jharenza) [reply content/07.supplemental.md:16]: Thanks - 98 samples from 13 patients seems like a high number of samples to me. I may hold on this until that PR is in.

I did some fuzzy math, and checked some of these numbers - we might want to be sure we have matches on at least `tumor_descriptor`. Anything below with N = 1 has a sample only in WGS only, and this accounts for 11 samples out of 52 in the `bs` dataframe below, which is much less than the 98 above.

```r
> matched_participants <- v21 %>%
+   filter(experimental_strategy != "RNA-Seq") %>%
+   group_by(Kids_First_Participant_ID) %>%
+   summarize(strategies = paste0(experimental_strategy, collapse = ",")) %>%
+   filter(grepl("WXS", strategies) & grepl("WGS", strategies)) %>%
+   pull(Kids_First_Participant_ID) 

> # Get the biospecimen IDS for these participants.
> bs <- v21 %>%
+   filter(Kids_First_Participant_ID %in% matched_participants) %>%
+   filter(experimental_strategy != "RNA-Seq" & !is.na(pathology_diagnosis)) %>%
+   select(Kids_First_Participant_ID, Kids_First_Biospecimen_ID, tumor_descriptor, experimental_strategy) %>%
+   unique() %>%
+   mutate(pt_desc = paste(Kids_First_Participant_ID, tumor_descriptor, sep = "_")) %>%
+   group_by(pt_desc, experimental_strategy) %>%
+   tally()

> as.data.frame(table(bs$pt_desc))
                                          Var1 Freq
1                PT_0MXPTTM3_Initial CNS Tumor    2
2                PT_1E3E6GMF_Initial CNS Tumor    2
3                PT_9GKVQ9QS_Initial CNS Tumor    2
4                PT_HGM20MW7_Initial CNS Tumor    2
5                PT_KBFM551M_Initial CNS Tumor    2
6  PT_KBFM551M_Progressive Disease Post-Mortem    1
7                PT_KTRJ8TFY_Initial CNS Tumor    2
8  PT_KTRJ8TFY_Progressive Disease Post-Mortem    1
9                PT_KZ56XHJT_Initial CNS Tumor    2
10                     PT_KZ56XHJT_Progressive    2
11 PT_KZ56XHJT_Progressive Disease Post-Mortem    1
12               PT_M23Q0DC3_Initial CNS Tumor    2
13               PT_M9XXJ4GR_Initial CNS Tumor    2
14               PT_NK8A49X5_Initial CNS Tumor    2
15                     PT_NK8A49X5_Progressive    2
16               PT_QA9WJ679_Initial CNS Tumor    2
17               PT_VPEMAQBN_Initial CNS Tumor    2
18               PT_WGVEF96B_Initial CNS Tumor    2
```

Comment (sjspielman) [reply content/07.supplemental.md:16]: I calculated those numbers based on the code that is used to create the figure itself, so we can chat about it over in that PR, and whether the figures themselves are accurate.

Comment (jharenza) [reply content/07.supplemental.md:16]: sounds good!

**Pull request merged**